ci-account-setup/doc/features.md

```diff
@@ -0,0 +1,41 @@
+# Features of th AWS CI Account Setup
+
+The AWS CI Account Setup provisions an AWS account for running tests from CI securely.
+
+## Create AWS resources from GitHub Actions
+
+`feat~create-aws-cresources-in-ci~1`
+
+We can create AWS resources from CI builds in out GitHub Actions CI builds.
```

Comment:
```suggestion
The AWS CI Account Setup can create AWS resources from CI builds in out GitHub Actions CI builds.
```

Reply:
The AWS CI Account Setup creates a user for CI. With the user we can create AWS resources from CI builds in our GitHub Actions CI builds.